Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6-alpine
+ENV PYTHONUNBUFFERED 1
```

Comment:
Please combine the `ENV` directives to limit the layers since these won't change and are related:

```
ENV PYTHONUNBUFFERED=1 \
        PY_ENV=development
```